cmd/kusk/examples/httpbin-spec.yaml

```diff
@@ -0,0 +1,224 @@
+swagger: "2.0"
```

Comment:
We already have this example manifest in other places
Should we keep this file?

Reply:
I don't think so